packages/turbo-workspaces/src/utils.ts

```diff
@@ -0,0 +1,119 @@
+import fs from "fs-extra";
+import path from "path";
+import glob from "fast-glob";
+import yaml from "js-yaml";
+import semver from "semver";
+import { PackageJson, Workspace } from "./types";
+import { ConvertError } from "./errors";
+
+// adapted from https://github.com/nodejs/corepack/blob/main/sources/specUtils.ts#L14
```

Comment:
Use `y` to get a permalink.
https://github.com/nodejs/corepack/blob/cae770694e62f15fed33dd8023649d77d96023c1/sources/specUtils.ts#L14

Reply:
TIL